lib/charms/mongodb/v1/shards_interface.py

```diff
@@ -295,7 +319,7 @@ def get_config_server_status(self) -> Optional[StatusBase]:
             return BlockedStatus(f"relation {REL_NAME} to shard not supported.")
 
         if not self.is_mongos_running():
-            return ErrorStatus("Internal mongos is not running.")
```

Comment:
it is actually not permitted to set ErrorStatus like we had before, it results in a broken charm and we get the following traceback:
```
  File "/var/lib/juju/agents/unit-shard-0/charm/venv/ops/model.py", line 2895, in status_set
    self._run('status-set', f'--application={is_app}', status, message)
  File "/var/lib/juju/agents/unit-shard-0/charm/venv/ops/model.py", line 2695, in _run
    raise ModelError(e.stderr)
ops.model.ModelError: ERROR invalid status "error", expected one of [maintenance blocked waiting active]
```